### PR TITLE
Update code owners

### PR DESCRIPTION
**Pull request by benashz**
- ignore scratch dir

### CODE REVIEW

**Comment by benashz** [comment]
@sarahethompson wanted to check to see if we still need these code owner directives?

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1 @@
-# release configuration
-
-/.release/                               @hashicorp/github-secure-vault-ecosystem
```
[the remark this answers is not in the document]
Comment (benashz) [reply CODEOWNERS:3]: @tomhjp confirmed with rel-eng we don't need to include them as code owners.

**Review by swenson — APPROVED** [review]
LGTM

**Comment by benashz** [comment]
Thanks!

File: CODEOWNERS (hunk 1 above)
[the remark this answers is not in the document]
Comment (tvoran) [reply CODEOWNERS:3]: Yeah rel-eng removed themselves from CODEOWNERS here [a few months ago](https://github.com/hashicorp/vault-k8s/pull/555), and I don't know of any reason we need to keep `github-secure-vault-ecosystem`. `hashicorp/vault-ecosystem` seems like a better fit for now.

**Pull request merged**